README.md

```diff
@@ -506,6 +510,60 @@ By default, the Klaviyo SDK automatically clears the badge count on app open. If
+* Imported `KlaviyoSwift` and `KlaviyoForms` SDK modules and adding it to the app target.
+
+### Setup
+
+Once you've created an in-app form in your Klaviyo account, to display it add the following code to your application.
+
+```swift
+    import KlaviyoSwift
+    import KlaviyoForms
+    ...
+
+    // Refer the behavior section for where to place this code.
+    KlaviyoSDK()
+        .initialize(with: "YOUR_KLAVIYO_PUBLIC_API_KEY")
+        .registerForInAppForms()
+
+    // if registering else where after `KlaviyoSDK` is initialized
+    KlaviyoSDK().registerForInAppForms()
+```
+
+### Behavior
+
+Once `registerForInAppForms()` is called, the SDK will load form data for your account and display no more than one form within 10 seconds,  based on form targeting and behavior settings.
+
+You can call `registerForInAppForms()` any time after initializing with your public API key to control when and where in your app's UI a form can appear. It is safe to register multiple times per application session. The SDK will internally prevent multiple forms appearing at once.
+
+Consider how often you want to register for forms. Below are some ideas on when forms can potentially be shown,
+
+
+| **App State**                | **Lifecycle Method**                              | **Example Implementation** |
+|------------------------------|--------------------------------------------------|------------------------------|
+| **App Launched (Cold Start)** | `application(_:didFinishLaunchingWithOptions:)` | [See here](https://github.com/klaviyo/klaviyo-swift-sdk/blob/master/Examples/KlaviyoSwiftExamples/Shared/AppDelegate.swift#L41) |
+| **App Became Active**         | `applicationDidBecomeActive(_:)`                 | [See here](https://github.com/klaviyo/klaviyo-swift-sdk/blob/master/Examples/KlaviyoSwiftExamples/Shared/AppDelegate.swift#L59) |
+| **Any App View Controller**         | `viewDidLoad()` | [See here](https://github.com/klaviyo/klaviyo-swift-sdk/blob/master/Examples/KlaviyoSwiftExamples/Shared/MenuPageViewController.swift#L35) |
+
+
+Registering from `applicationDidBecomeActive(_:)` is advisable as it increases the chance of your user seeing the form. However, be advised that this will be shown as soon as the form is ready in the SDK, so you may still need to condition this based on the user's context within your application. Future versions of this product will provide more control in this regard.
+
+
+**Note:** At this time, when device orientation changes any currently visible form is closed and will not be re-displayed automatically.
+
+### Deep linking
+
+Deep linking to a particular screen based on user action from an IAF is similar to handling deep links originating from push notifications. [Step 3](#step-3-implement-handling-deep-links-in-your-app) of the deep linking section outlines exactly how this can be achieved. Currently, if a web URL is included as the deep link, the SDK will open the URL in the default browser.
```

Comment:
> Currently, if a web URL is included as the deep link, the SDK will open the URL in the default browser

I don't think this is true for universal links, if you want to get into that level of detail anyway.

Reply:
By which I mean if the app is authorized to open universal links for a given web domain, then that should still open their app.

Reply:
Perhaps
> For further information on how the deep link is handled, see [apple documentation](https://developer.apple.com/documentation/uikit/uiapplication/open(_:options:completionhandler:)).